jenkins_rock/rockcraft.yaml

```diff
@@ -28,7 +28,7 @@ parts:
       mkdir -p $CRAFT_OVERLAY/var
       chmod 755 $CRAFT_OVERLAY/var
       groupadd -R $CRAFT_OVERLAY --gid 2000 jenkins
-      useradd -R $CRAFT_OVERLAY --system --gid 2000 --uid 2000 --home /srv/jenkins jenkins
+      useradd -R $CRAFT_OVERLAY --system --gid 2000 --uid 2000 --home /var/lib/jenkins jenkins
```

Comment:
In addition to this, I think you'll also want to update where the `jenkins.war` is stored?

Reply:
`jenkins.war` comes packaged in the rock image - if we do update the charm and the image, I think it might be better that it is not in a persistent location? WDYT?

Reply:
Ah sure, works for me. Thanks